pkg/diagnostics/v2/client_test.go

```diff
@@ -55,6 +59,43 @@ func TestEmptyList(t *testing.T) {
 	assert.Empty(t, bundles)
 }
 
+func TestCreateHappyPath(t *testing.T) {
+	var newID string
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		re := regexp.MustCompile("^/system/health/v1/diagnostics/([a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12})$")
+		urlMatch := re.FindStringSubmatch(r.URL.Path)
+		newID = urlMatch[1]
+		require.True(t, len(urlMatch) == 2, fmt.Sprintf("URL %s doesn't match %s", r.URL.Path, re.String()))
```

Comment:
Maybe simpler to rely on https://godoc.org/github.com/satori/go.uuid#FromString for validating the uuid.